docs/api.rst

```diff
@@ -3044,6 +3044,13 @@ Search for assets
                   "identifier": "eip155:1/erc20:0xB6eD7644C69416d67B522e20bC294A9a9B405B31",
                   "name": "0xBitcoin",
                   "symbol": "0xBTC"
+                  "chain": "ethereum",
```

Comment:
I think @kelsos wanted a value identifier here too, right kelsos? To have smaller data in the json?

Also the api docs should contain theinformation of the returned responseand explain each field. This seems to be missing. It should especially show that chain can be null. And again question here for @kelsos. I think he has mentioned that it's much faster/better for them to just omit an attribute if it's null. As the response has a smaller size

Reply:
For the time being, we can do that and create a new issue to optimize.

It would be preferable to omit null values since it reduces the response size and makes type handling easier. Missing properties are treated as undefined (optional), on the other hand, null is defined and it is an object so we have to account for that in the type :|

Reply:
okey got it. Is no effort to change it and use the numeric value. I believe you already have a mapping since is the same used for the identifiers in the assets

Reply:
After discussing this in discord we said that we can worry about optimization later and how to generalize this field for other chains. For now we would use the evm chain string value

Reply:
We discussed in discord and this is pushed for later.